app/index.js

```diff
@@ -142,6 +142,13 @@ module.exports = yeoman.Base.extend({
       pkg.name = options.projectName;
       pkg.description = options.projectDescription;
 
+      if (!pkg['scripts'] && options['themePath']) {
+        pkg.scripts = {};
+      }
+      if (!pkg.scripts['postinstall'] && options['themePath']) {
```

Comment:
This is causing an error in the tests because pkg.scripts can be undefined if options['themePath'] is not true. See: https://travis-ci.org/phase2/generator-gadget/jobs/113321948#L1435